x/pairing/keeper/staking.go

```diff
@@ -12,6 +13,8 @@ import (
 	spectypes "github.com/lavanet/lava/x/spec/types"
 )
 
+const MAX_CHANGE_RATE = 1
```

Comment:
why is this here and not in the types folder somewhere (we usually put constants in types.go)

Reply:
these are only related to this file

Reply:
https://github.com/lavanet/lava/pull/1263/commits/5cda7df3147294fb3b1aaa960ff157f5d8e2b00b